Integrations/DomainTools_Iris/DomainTools_Iris.py

```diff
@@ -0,0 +1,646 @@
+    alexa_rank = domain_result.get('alexa')
+    tags = domain_result.get('tags')
+
+    registrant_name = domain_result.get('registrant_name').get('value')
+    registrant_org = domain_result.get('registrant_org').get('value')
+    domain_registrant_contact = {
+        'Country': domain_result.get('registrant_contact').get('country').get('value'),
+        'Email': ', '.join([x.get('value') for x in domain_result.get('registrant_contact').get('email')]),
+        'Name': domain_result.get('registrant_contact').get('name').get('value'),
+        'Phone': domain_result.get('registrant_contact').get('phone').get('value'),
+    }
+    registrant_contact = {
+        'Country': domain_result.get('registrant_contact').get('country'),
+        'Email': domain_result.get('registrant_contact').get('email'),
+        'Name': domain_result.get('registrant_contact').get('name'),
+        'Phone': domain_result.get('registrant_contact').get('phone'),
+    }
+    admin_contact = {
+        'Country': domain_result.get('admin_contact').get('country'),
+        'Email': domain_result.get('admin_contact').get('email'),
+        'Name': domain_result.get('admin_contact').get('name'),
+        'Phone': domain_result.get('admin_contact').get('phone'),
+    }
+    technical_contact = {
+        'Country': domain_result.get('technical_contact').get('country'),
+        'Email': domain_result.get('technical_contact').get('email'),
+        'Name': domain_result.get('technical_contact').get('name'),
+        'Phone': domain_result.get('technical_contact').get('phone'),
+    }
+
+    billing_contact = {
+        'Country': domain_result.get('billing_contact').get('country'),
+        'Email': domain_result.get('billing_contact').get('email'),
+        'Name': domain_result.get('billing_contact').get('name'),
+        'Phone': domain_result.get('billing_contact').get('phone'),
+    }
+    soa_email = [x.get('value') for x in domain_result.get('soa_email')]
+    ssl_email = [x.get('value') for x in domain_result.get('ssl_email')]
+    email_domains = [x.get('value') for x in domain_result.get('email_domain')]
+    additional_whois_emails = domain_result.get('additional_whois_email')
+    domain_registrant = domain_result.get('registrar').get('value') if isinstance(domain_result.get('registrar'),
+                                                                                  dict) else domain_result.get(
+        'registrar')
+    registrar_status = domain_result.get('registrar_status')
+    ip_data = domain_result.get('ip')
+    ip_country_code = domain_result.get('ip')[0] if len(domain_result.get('ip')) else ''
+    mx_servers = domain_result.get('mx')
+    spf_info = domain_result.get('spf_info')
+    ssl_certificates = domain_result.get('ssl_info')
+    redirects_to = domain_result.get('redirect')
+
+    domain_tools_context = {
+        'Name': domain,
+        'LastEnriched': datetime.now().strftime('%Y-%m-%d'),
```

Comment:
are we sure the string date format will always be this ?

Reply:
We are creating so we should know.